### PR TITLE
FUSETOOLS-3327 - upgrade internal wsdl2rest version

### PR DESCRIPTION
**Pull request by apupier**
Signed-off-by: Aurélien Pupier <apupier@redhat.com>

note versiosn of other dependencies can be found in this pom https://maven.repository.redhat.com/ga/org/jboss/fuse/wsdl2rest/wsdl2rest/0.8.0.fuse-760026-redhat-00001/wsdl2rest-0.8.0.fuse-760026-redhat-00001.pom

### CODE REVIEW

**Comment by apupier** [comment]
it failed several times already with:

`18:06:25 org.apache.maven.lifecycle.LifecycleExecutionException: Failed to execute goal on project org.fusesource.ide.wsdl2rest.ui: Could not resolve dependencies for project org.fusesource.ide.editor:org.fusesource.ide.wsdl2rest.ui:eclipse-plugin:11.6.0-SNAPSHOT: Could not find artifact org.eclipse.jetty:jetty-continuation:jar:9.4.25.v20191220 in central (https://repository.jboss.org/nexus/content/repositories/central/)`

but the jar is available there https://repository.jboss.org/nexus/content/repositories/central/org/eclipse/jetty/jetty-continuation/9.4.25.v20191220/ and it is even available on maven central https://repository.jboss.org/nexus/content/repositories/central/org/eclipse/jetty/jetty-continuation/9.4.25.v20191220/

**Comment by bfitzpat** [comment]
looks like it passes in jenkins at least partially?

**Comment by apupier** [comment]
> looks like it passes in jenkins at least partially?

there is still the same build error, again and again in the last builds

**Comment by bfitzpat** [comment]
> there is still the same build error, again and again in the last builds

Weird. Wonder if something is up with nexus access from jenkins?

**Comment by apupier** [comment]
I reproduce the error if I exclude Maven central repository in pom of the Eclipse Fuse Tooling plugins.
I don't reproduce if I build the project as-is. (for sure needs to clean the local .m2 to be in same state)

I'm unable to reproduce with a simpler project https://github.com/apupier/jetty-continuation-download-issue

**Comment by apupier** [comment]
It is now working.
Seems the issue is that repository.jboss.org has intermittent failure which was occuring most of the time with new versions of jetty-continuation.
Deactivating the repository for the wsdl2rest UI bundles ddid the trick

**Comment by apupier** [comment]
(fuse PR still marked as pending but it is the time to "finish" the job, the build is in success)

File: editor/plugins/org.fusesource.ide.wsdl2rest.ui/.classpath [hunk 1]
```diff
@@ -21,6 +20,8 @@
 	<classpathentry exported="true" kind="lib" path="libs/commons-digester.jar"/>
 	<classpathentry exported="true" kind="lib" path="libs/commons-logging.jar"/>
 	<classpathentry exported="true" kind="lib" path="libs/commons-validator.jar"/>
+	<classpathentry exported="true" kind="lib" path="libs/commons-lang.jar"/>
```
Comment (lhein) [inline editor/plugins/org.fusesource.ide.wsdl2rest.ui/.classpath:23]: why did we need to add commons-lang in addition to commons-lang3 ?

Comment (apupier) [reply editor/plugins/org.fusesource.ide.wsdl2rest.ui/.classpath:23]: This .classpath was updated to stay coherent with what is described in build.properties. It was missing from when the dependencies were added previously

Comment (apupier) [reply editor/plugins/org.fusesource.ide.wsdl2rest.ui/.classpath:23]: build.properties:
https://github.com/jbosstools/jbosstools-fuse/blob/6dade9464924f3f427e9af9a4b19f6b7d765086b/editor/plugins/org.fusesource.ide.wsdl2rest.ui/build.properties#L104-L105

previous commit adding commons-lang and commons-lang3 to build.properties but only commons-lang3 to .classpath: https://github.com/jbosstools/jbosstools-fuse/commit/35d9f20123022fbc8ce9a2bf3e94f2853ff4c877

**Pull request merged**